crates/datafusion_ext/src/metrics.rs

```diff
@@ -212,7 +212,7 @@ pub struct AggregatedMetrics {
     /// Total time taken for a plan to execute.
     pub elapsed_compute_ns: u64,
     /// Total bytes processed.
-    pub bytes_processed: u64,
+    pub bytes_read: u64,
```

Comment:
While we're here, lets go ahead an rename the above gauge name too.

Reply:
Updated.